server-common/src/main/resources/jcasbin_model.conf

```diff
@@ -0,0 +1,50 @@
+; Licensed to the Apache Software Foundation (ASF) under one
+; or more contributor license agreements.  See the NOTICE file
+; distributed with this work for additional information
+; regarding copyright ownership.  The ASF licenses this file
+; to you under the Apache License, Version 2.0 (the
+; "License"); you may not use this file except in compliance
+; with the License.  You may obtain a copy of the License at
+;
+;  http://www.apache.org/licenses/LICENSE-2.0
+;
+; Unless required by applicable law or agreed to in writing,
+; software distributed under the License is distributed on an
+; "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+; KIND, either express or implied.  See the License for the
+; specific language governing permissions and limitations
+; under the License.
+
+; The model file of jcasbin defines a permission model used to describe the permissions
+; of users, groups, and roles for a single metadata type.
+
+[request_definition]
+r = sub,metalakeId,metadataType,metadataId, act
+
+; "r" represents the parameters passed when making a request to jCasbin.
+; "sub" represents a role, user, or groupCode.
+; "metalakeId" represents the metalake to which a role, user, or group belongs.
+; "metadataType" represents the type of metadata.
+; "metadataId" represents the id of metadata
+; "act" represents the privilege that needs to be authorized or whether it is an OWNER.
+
+[policy_definition]
+p = sub,metalakeId,metadataType,metadataId, act, eft
+
+; "p" represents a permission policy.
+; "eft" stands for "effect" which can be either "allow" or "deny".
+
+[role_definition]
+g = _, _
+
+; "g" represents the membership or ownership relationship of users, groups, or roles.
+
+[policy_effect]
+e = some(where (p.eft == allow)) && !some(where (p.eft == deny))
+
+; "e" represents the effect of the "eft"
```

Comment:
I think we can add detailed comments here, maybe just like:
1. `some(where (p.eft == allow))`: What mean is this code?
2. `!some(where (p.eft == deny))`: What mean is this code?

and other places.